<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>610</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="list_titel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>581</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20pt &quot;Calibri&quot;;
</string>
   </property>
   <property name="text">
    <string>Order for: Naor</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QFrame" name="order_frame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>580</width>
     <height>80</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="order_dates">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>251</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 16pt &quot;Calibri&quot;;</string>
    </property>
    <property name="text">
     <string>01/01/2024 - 20/01/2024</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="order_guests">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>10</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 16pt &quot;Calibri&quot;;</string>
    </property>
    <property name="text">
     <string>5 Guests</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="order_view_btn">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>20</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(13, 153, 255);
border-radius:20px;
font-size:20px;
</string>
    </property>
    <property name="text">
     <string>View order</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="order_frame_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>580</width>
     <height>80</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="order_dates_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>251</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 16pt &quot;Calibri&quot;;</string>
    </property>
    <property name="text">
     <string>01/02/2024 - 20/02/2024</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="order_guests_2">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>10</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 16pt &quot;Calibri&quot;;</string>
    </property>
    <property name="text">
     <string>2 Guests</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="order_view_btn_2">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>20</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(13, 153, 255);
border-radius:20px;
font-size:20px;
</string>
    </property>
    <property name="text">
     <string>View order</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
